elements/pf-code-block/test/pf-code-block.spec.ts

```diff
@@ -43,34 +43,34 @@ const expandElementByDefault = html`
       connectionConfig:
       url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
     </script>
-    <pfe-clipboard slot="actions" copy-from="#expandable-code-expanded"></pfe-clipboard>
-  </pfe-code-block>
+    <pf-clipboard slot="actions" copy-from="#expandable-code-expanded"></pf-clipboard>
```

Comment:
`<pf-clipboard ...></pf-clipboard>` should be `<pf-clipboard-copy ...></pf-clipboard-copy>`